appserver/web/war-util/src/main/java/org/glassfish/web/loader/WebappClassLoader.java

```diff
@@ -179,7 +179,7 @@ public class WebappClassLoader extends GlassfishUrlClassLoader
      * Use this variable to invoke the security manager when a resource is
      * loaded by this classloader.
      */
-    private final boolean packageDefinitionEnabled = System.getProperty("package.definition") != null;
+    private final boolean packageDefinitionEnabled = Boolean.getBoolean("package.definition");
```

Comment:
Isn't the value obtained using "package.definition" a list of package names, not a boolean string?

Reply:
Nice catch, I tried to search the string and it seems to me it is even worse for a long time ... the property is set by catalina not to System properties, but via java.security.Security.setProperty. So this was always false.

Reply:
Nice finding. I believe it deserves separate issue, that could be solved separately in dedicated PR.

Reply:
I brought back the original System.getProperty. Even if it is not probably worth of merging to older branches and SecurityManager will die soon, GF should be correct at the moment, so after merging this I can create another tiny PR :)